benchmarks/bench_sm120_direct_fused_moe.py

```
@@ -0,0 +1,181 @@
+    output = torch.empty(max_tokens, hidden, dtype=torch.bfloat16, device="cuda")
+    cutlass_output = torch.empty_like(output)
+    workspace = sm120_direct_fused_moe_workspace(
+        max_tokens, topk, intermediate, device="cuda"
+    )
+
+    rows = []
+    for num_tokens in range(1, 9):
+
+        def run_direct():
+            return sm120_direct_fused_moe(
+                hidden_states[:num_tokens],
```

Ruff (0.16.0)

[warning] Function definition does not bind loop variable `num_tokens`

(B023)